### ISSUE TITLE
Intellij is slow and laggy

### ISSUE THREAD

**Issue by karrgozeyan** [qtile/qtile#3657]
### The issue:

Intellij in qtile is slow and laggy. These problems things annoy me the most: 

- The characters I type appear with some delay (about 0.5 second).
- If I move cursor with the mouse it happens with the same delay.
- Selecting some block of code is laggy as well.

Here is what I have tired:

- set wmname to LG3D
- set environmental variable _JAVA_AWT_WM_NONREPARENTING to 1

Any idea how can I fix this?

NOTE: I have also DWM installed alogside and it works perfectly on it.
Qtile version is 0.21.0
There are no relevant logs
### Required:

- [X] I have searched past issues to see if this bug has already been reported.

**Comment by m-col** [comment]
Please could you post an `xprop` of the window in qtile and dwm?

**Comment by karrgozeyan** [comment]
Xprop on Qtile:
```
_MOTIF_DRAG_RECEIVER_INFO(_MOTIF_DRAG_RECEIVER_INFO) = 0x6c, 0x0, 0x5, 0x0, 0x4e, 0x0, 0x20, 0x2, 0x0, 0x0, 0x0, 0x0, 0x10, 0x0, 0x0, 0x0
XdndAware(ATOM) = BITMAP
_NET_FRAME_EXTENTS(CARDINAL) = 4, 4, 4, 4
_NET_WM_DESKTOP(CARDINAL) = 3
WM_STATE(WM_STATE):
		window state: Normal
		icon window: 0x0
_NET_WM_USER_TIME(CARDINAL) = 9620019
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_NORMAL
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x3, 0x1, 0x1, 0x0, 0x0
_NET_WM_STATE(ATOM) = _NET_WM_STATE_MAXIMIZED_VERT, _NET_WM_STATE_MAXIMIZED_HORZ
WM_HINTS(WM_HINTS):
		Client accepts input or input focus: False
		Initial state is Normal State.
_NET_WM_ICON(CARDINAL) = 	Icon (128 x 128):
	(not shown)
	Icon (32 x 32):
	(not shown)

_NET_WM_PID(CARDINAL) = 42734
WM_CLIENT_MACHINE(STRING) = "archlinux"
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS
WM_CLASS(STRING) = "jetbrains-idea", "jetbrains-idea"
WM_CLIENT_LEADER(WINDOW): window id # 0x2200008
_NET_WM_ICON_NAME(UTF8_STRING) = "thread-factoring – ThreadUnionImpl.java"
WM_ICON_NAME(COMPOUND_TEXT) = "thread-factoring – ThreadUnionImpl.java"
_NET_WM_NAME(UTF8_STRING) = "thread-factoring – ThreadUnionImpl.java"
WM_NAME(COMPOUND_TEXT) = "thread-factoring – ThreadUnionImpl.java"
WM_NORMAL_HINTS(WM_SIZE_HINTS):
		user specified location: 0, 28
		program specified location: 0, 28
		program specified size: 1920 by 1052
		program specified minimum size: 340 by 0
		window gravity: NorthWest
```
Xprop on DWM
```
_MOTIF_DRAG_RECEIVER_INFO(_MOTIF_DRAG_RECEIVER_INFO) = 0x6c, 0x0, 0x5, 0x0, 0x50, 0x0, 0xe0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x10, 0x0, 0x0, 0x0
XdndAware(ATOM) = BITMAP
WM_STATE(WM_STATE):
		window state: Normal
		icon window: 0x0
_NET_WM_USER_TIME(CARDINAL) = 10476036
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_NORMAL
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x3, 0x1, 0x1, 0x0, 0x0
_NET_WM_STATE(ATOM) = _NET_WM_STATE_MAXIMIZED_VERT, _NET_WM_STATE_MAXIMIZED_HORZ
WM_HINTS(WM_HINTS):
		Client accepts input or input focus: False
		Initial state is Normal State.
_NET_WM_ICON(CARDINAL) = 	Icon (128 x 128):
	(not shown)
	Icon (32 x 32):
	(not shown)

_NET_WM_PID(CARDINAL) = 51628
WM_CLIENT_MACHINE(STRING) = "archlinux"
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS
WM_CLASS(STRING) = "jetbrains-idea", "jetbrains-idea"
WM_CLIENT_LEADER(WINDOW): window id # 0xe00008
_NET_WM_ICON_NAME(UTF8_STRING) = "thread-factoring – ThreadUnionImpl.java"
WM_ICON_NAME(COMPOUND_TEXT) = "thread-factoring – ThreadUnionImpl.java"
_NET_WM_NAME(UTF8_STRING) = "thread-factoring – ThreadUnionImpl.java"
WM_NAME(COMPOUND_TEXT) = "thread-factoring – ThreadUnionImpl.java"
WM_NORMAL_HINTS(WM_SIZE_HINTS):
		user specified location: 0, 28
		program specified location: 0, 28
		program specified size: 1920 by 1052
		program specified minimum size: 340 by 0
		window gravity: NorthWest
```

I also checked for AwesomeWM and Xmonad. It works fine there as well.

**Comment by karrgozeyan** [comment]
Should I close this issue? It seems there no solution at this moment.

**Comment by m-col** [comment]
Leave it at the moment, it likely reflects a bug in Qtile that needs fixing

**Comment by karrgozeyan** [comment]
Today I decided to give it another try on a freshly installed Arch and it is working fine.
I assume the problem was solved by the latest update of Intellij (2022.1.3-1) and Jetbrains Runtime (2022.1.3-1). I think the main problem was namely Jetbrains Runtime.

**Comment by m-col** [comment]
OK, that's great to hear!
